lib/internal/test_runner/harness.js

```diff
@@ -142,8 +142,8 @@ function setup(root) {
   const rejectionHandler =
     createProcessEventHandler('unhandledRejection', root);
   const coverage = configureCoverage(root, globalOptions);
-  const exitHandler = async () => {
-    await root.run(new ERR_TEST_FAILURE(
+  const exitHandler = () => {
+    root.postRun(new ERR_TEST_FAILURE(
```

Comment:
why not await?

Reply:
@rluvaton `postRun` returns `void` (and not a `Promise`)

Reply:
I thought I changed that (or I'm thinking of another function there) 😅